src/CollectBlock.ts

```diff
@@ -198,7 +197,7 @@ export class CollectBlock {
   constructor (bot: Bot) {
     this.bot = bot
     this.targets = new Targets(bot)
-    this.movements = new Movements(bot, mcDataLoader(bot.version))
+    this.movements = new Movements(bot)
```

Comment:
Why that change

Reply:
I was getting a TypeScript error with it

Reply:
![image](https://github.com/PrismarineJS/mineflayer-collectblock/assets/47758296/e5294c3c-fd0c-4d57-8e43-eed981d548a7)